hildon/hildon-gtk.h

```diff
@@ -18,6 +18,55 @@
 #define                                         __HILDON_GTK_H__
 
 #include                                        <gtk/gtk.h>
+/// Temporary hack to provide the types from the Maemo fork of gtk
+/// This was copy-pasted from hildonize-gtk-enums.diff from maemo-leste/gtk
+#ifndef MAEMO_GTK
+typedef enum
+{
+  GTK_INVALID_INPUT_MAX_CHARS_REACHED,
+  GTK_INVALID_INPUT_MODE_RESTRICTION
+} GtkInvalidInputType;
+
+typedef enum
+{
+  HILDON_GTK_INPUT_MODE_ALPHA             = 1 << 0,
+  HILDON_GTK_INPUT_MODE_NUMERIC           = 1 << 1,
+  HILDON_GTK_INPUT_MODE_SPECIAL           = 1 << 2,
+  HILDON_GTK_INPUT_MODE_HEXA              = 1 << 3,
+  HILDON_GTK_INPUT_MODE_TELE              = 1 << 4,
+  HILDON_GTK_INPUT_MODE_FULL              = (HILDON_GTK_INPUT_MODE_ALPHA | HILDON_GTK_INPUT_MODE_NUMERIC | HILDON_GTK_INPUT_MODE_SPECIAL),
+  HILDON_GTK_INPUT_MODE_NO_SCREEN_PLUGINS = 1 << 27,
+  HILDON_GTK_INPUT_MODE_MULTILINE         = 1 << 28,
+  HILDON_GTK_INPUT_MODE_INVISIBLE         = 1 << 29,
+  HILDON_GTK_INPUT_MODE_AUTOCAP           = 1 << 30,
+  HILDON_GTK_INPUT_MODE_DICTIONARY        = 1 << 31
+} HildonGtkInputMode;
+
+/* Temporary compatibility define */
```

Comment:
Same.

Reply:
This is not my comment, it was taken directly from [this patch](https://github.com/maemo-leste/gtk/blob/552c94ce4a423dfc5fe775dd71f55388be7b2d8d/debian/patches/hildonize-gtk-enums.diff#L28). If you remove this comment from maemo-leste/gtk, I'll remove it from here.

Reply:
OK, let's leave this.